plugins/github-actions/src/components/WorkflowRunDetails/WorkflowRunDetails.tsx

```diff
@@ -175,14 +175,17 @@ export const WorkflowRunDetails = ({ entity }: { entity: Entity }) => {
   const jobs = useWorkflowRunJobs({ hostname, owner, repo });
 
   const error = projectName.error || (projectName.value && details.error);
+  const loading = projectName.loading || details.loading || !details.value;
+
   const classes = useStyles();
-  if (error) {
+
+  if (error && error.message) {
```

Comment:
Is this the real issue here, that error is set to something, but it's false positive? It's something that isn't a real error?

Reply:
Tbh we should fix the `Promise.reject('...')` of strings being thrown around. Just like `throw`, promise rejections should always come in the form on an `Error`.